phantasm-at-rest/src/test/java/com/chiralbehaviors/CoRE/phantasm/resources/ResourcesTest.java

```diff
@@ -152,7 +152,7 @@ public void testGraphQlCreateAndMutate() throws Exception {
                                            .toString());
         variables.put("name", "hello");
         variables.put("description", "goodbye");
-        QueryRequest request = new QueryRequest("mutation m ($name: String, $description: String, $artifact: String) { CreateThing1(state: { setName: $name, setDescription: $description, setDerivedFrom: $artifact}) { id name } }",
+        QueryRequest request = new QueryRequest("mutation m ($name: String!, $description: String, $artifact: String) { CreateThing1(state: { setName: $name, setDescription: $description, setDerivedFrom: $artifact}) { id name } }",
```

Comment:
When you have an argument declared "non null" in the schema, the argument to the query must also be declared "not null".  The ! indicates "not null"